docs/pages/repo/docs/reference/configuration.mdx

```diff
@@ -176,6 +187,27 @@ The list of environment variables a task depends on.
   caching](/repo/docs/core-concepts/caching#automatic-environment-variable-inclusion).
 </Callout>
 
+## `experimentalPassthroughEnv`
+
+`type: string[]`
+
+An allowlist of environment variables that should be made available to this task but should not contribute to the task's cache key. Using this key opts this task into strict environment variable mode.
```

Comment:
Maybe add a note that the _names_ contribute to the hash key, but the _values_ don't. So updating this list will bust the cache, but changing the values of vars in the list won't